### PR TITLE
Verify CocoaPods

### PR DESCRIPTION
**Pull request by teobugslayer**
See https://github.com/NativeScript/nativescript-cli/issues/1294

How it works:

- install iOS runtime into a temporary directory
- create a simple Podfile into the template project
- run (sandobox-)pod
- verify nothing breaks

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
hm, why should we create new project, add platform and verify `pod install` command after that. Do we want to validate that project with pods can be created or that pods are useful on the machine?
I prefer to create Podfile in a temp dir and call `pod install`. Could this be done or we need some files from the runtime?

**Comment by teobugslayer** [comment]
Well, CocoaPods need a Xcode project to work on and that was the easiest way I could think of creating one.

**Comment by rosen-vladimirov** [comment]
We can print dots while waiting for pod install (we have a method in helpers that does this :) )

**Comment by teobugslayer** [comment]
@rosen-vladimirov, without existing {N} project, CocoaPods complains with `[!] Could not automatically select an Xcode project.`

**Comment by rosen-vladimirov** [comment]
Can we mock it - for example we can add one default proj file in our resources, copy it to temp folder, create Podfile there and call `pod install`
There are many, many reasons why `tns create app1 && tns platform add ios` may fail and it will break doctor command. Do we want this?

File: lib/services/doctor-service.ts [hunk 1]
```diff
@@ -83,5 +97,35 @@ class DoctorService implements IDoctorService {
...
 		}
 	}
+
+	private verifyCocoaPods(): boolean {
+		this.$logger.out("Verifying CocoaPods. This may take more than a minute, please be patient.");
+
+		let temp = require("temp");
+		temp.track();
+		let projDir = temp.mkdirSync("nativescript-check-cocoapods");
+		this.$npm.install("tns-ios", projDir).wait();
```
Comment (rosen-vladimirov) [inline lib/services/doctor-service.ts]: maybe we should try-catch this one as well. As we know - npm fails sometimes and in this case our doctor command will fail

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**